src/distilabel/steps/generators/huggingface.py

```diff
@@ -567,8 +570,13 @@ def load(self) -> None:
                 keep_in_memory=self.keep_in_memory,
                 storage_options=self.storage_options,
             )
-            if self.config:
-                ds = ds[self.config]
+            if self.config not in ds.keys():
+                # TODO: Add FAQ for this case, pointing to the Distiset documentation on configurations.
+                raise ValueError(
+                    f"Configuration '{self.config}' not found in the Distiset, available ones"
+                    f" are: {list(ds.keys())}."
+                )
```

Comment:
Maybe we can add an additional phrase guiding the users on what they have to do to potentially fix the issue:

"... Please, try changing `config` attribute/runtime parameter value to one of the valid configurations"

Reply:
I've updated the error message to be more informative, but this is a candidate for a refactor after https://github.com/argilla-io/distilabel/pull/911